hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/annotations/InterimSPI.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.evm.annotations;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Marks an _interim_ SPI interface or class, extensible _outside_ of the `hedera-evm` module
+ *
+ * "Interim" because there's a bit of redesign needed here to find proper extension points
+ * of this module.  This will mark classes suitable for being extended in the meantime.
+ */
+@Target(TYPE)
+@Retention(SOURCE)
+public @interface InterimSPI {}
```

Comment:
You should probably specific whether this annotation is or is not inherited.

Reply:
See: https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/Inherited.html

Reply:
hmm, let me read up on that ...

Reply:
done!